samples/chat/csharp-ooproc/ChatBot.cs

```diff
@@ -51,31 +49,26 @@ public class CreateChatBotOutput
         [AssistantCreateOutput()]
         public AssistantCreateRequest? ChatBotCreateRequest { get; set; }
 
-        public HttpResponseData? HttpResponse { get; set; }
+        [HttpResult]
+        public IActionResult? HttpResponse { get; set; }
     }
 
     [Function(nameof(PostUserResponse))]
-    public static async Task<HttpResponseData> PostUserResponse(
+    public static async Task<IActionResult> PostUserResponse(
         [HttpTrigger(AuthorizationLevel.Function, "post", Route = "chats/{chatId}")] HttpRequestData req,
         string chatId,
         [AssistantPostInput("{chatId}", "{Query.message}", Model = "%CHAT_MODEL_DEPLOYMENT_NAME%")] AssistantState state)
     {
-        HttpResponseData response = req.CreateResponse(HttpStatusCode.OK);
-
-        response.Headers.Add("Content-Type", "text/plain");
-        await response.WriteStringAsync(state.RecentMessages.LastOrDefault()?.Content ?? "No response returned.");
-        return response;
+        return new OkObjectResult(state.RecentMessages.LastOrDefault()?.Content ?? "No response returned.");
```

Comment:
any way to await on these calls?
for both chat and assistant samples returning state.